### PR TITLE
React routing

### PR DESCRIPTION
**Pull request by ganhari123**
React routing completed may need to come back to this though

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -63,6 +63,6 @@ function normalizePort(val) {
 // Set up the port
 let port = normalizePort(process.env.PORT || '3000');
 
-server.listen(port, () => {
+server.listen(3000, () => {
```
Comment (nareddyt) [inline server.js]: Needs to be `port` so it works on heroku

File: src/signupTutor.js [hunk 2]
```diff
@@ -64,89 +64,17 @@ class SignupTutor extends React.Component {
...
-                    type="submit"
-                    value="SUBMIT" />
-                </div>
-                <div className="row col-xs-12">
-                    <h5 className="signup-dialogue">Already have an account? <a className="signup-anchor" href='/home/login'>Click here to log in!</a></h5>
-                </div>
-            </form>
-        </div>
-        );
+    if (currentConfirmPassword === this.state.password) {
```
Comment (nareddyt) [inline src/signupTutor.js]: Lol idk what happened here, but it seems like we lost of bunch of stuff? Is this file not used anymore maybe? I remember my branch `signup-validation` has a file called SignUpTutor (with caps)

Comment (ganhari123) [reply src/signupTutor.js]: Who did the sign up validation?

File: server.js (hunk 1 above)
Comment (ganhari123) [reply server.js]: Changed

**Pull request merged**